Fraudio/index.ts

```diff
@@ -0,0 +1,48 @@
+import * as selectively from "selectively"
+import * as authly from "authly"
+
+export interface Fraudio {
+	score: number
```

Comment:
Original value is actually a string containing a number
```suggestion
	score: string
```

Reply:
I'll convert to number in the from function.